kogito-kafka-quickstart-springboot/README.md

```diff
@@ -59,6 +110,13 @@ java -jar target/kogito-kafka-quickstart-sprintboot-{version}.jar
 
 To make use of this application it is as simple as putting a message on `travellers` topic with following content 
 
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
+
+Content
+
```

Comment:
This content does not work, it should be a full one, like in Quarkus:

```
{
  "specversion": "0.3",
  "id": "21627e26-31eb-43e7-8343-92a696fd96b1",
  "source": "",
  "type": "TravellersMessageDataEvent_3",
  "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]",
  "data": { 
	"firstName" : "Jan", 
	"lastName" : "Kowalski", 
	"email" : "jan.kowalski@example.com", 
	"nationality" : "Polish"
	}
}
```

Reply:
and please put the json as a one-line, as the user may want to copy/paste it

Reply:
Fixed @radtriste